fetch_api/src/fetch_api/arm.py

```diff
@@ -76,6 +76,19 @@ def __init__(self):
         self._move_group_client.wait_for_server(rospy.Duration(10))
         self._compute_ik = rospy.ServiceProxy('compute_ik', GetPositionIK)
         self._tf_listener = TransformListener()
+        self.move_group = MoveGroupInterface(ARM_GROUP_NAME, "base_link")
```

Comment:
I'm not sure the value of using MoveGroupInterface given the class already wraps the MoveGroup server. The interface spins up a bunch more action clients than we need: http://docs.ros.org/melodic/api/moveit_ros_planning_interface/html/move__group__interface_8cpp_source.html.